transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoopGroup.java

```diff
@@ -119,7 +127,10 @@ public void setIoRatio(int ioRatio) {
 
     @Override
     protected EventLoop newChild(Executor executor, Object... args) throws Exception {
+        final RejectedExecutionHandler rejectedExecutionHandler = args.length >= 3 ?
```

Comment:
should we instead just make sure we always pass `RejectedExecutionHandlers.reject()` to the super constructor upfront?